Dockerfile

```diff
@@ -4,7 +4,7 @@
 #
 
 # Building the Speculos environment
-FROM ghcr.io/ledgerhq/speculos-builder:latest AS builder
+FROM ledger/speculos-builder:latest AS builder
```

Comment:
Why?

Reply:
Actually, I have had to generate a new speculos-builder container compatible with my MAC M1. I can publish this specific container into `ghcr.io/ledgerhq`. In any case, trying to build the speculos container by pulling `ghcr.io/ledgerhq/speculos-builder` fails....

Reply:
Please delete this change, it breaks the CI

Reply:
Ok